modules/ipp/scanner.go

```diff
@@ -431,28 +524,28 @@ func (scanner *Scanner) Grab(scan *scan, target *zgrab2.ScanTarget, version *ver
 		}
 		if strings.HasPrefix(strings.ToUpper(p), "CUPS/") {
 			scan.results.CUPSVersion = p
-			err := scanner.augmentWithCUPSData(scan, target, version)
-			if err != nil {
-				log.WithFields(log.Fields{
-					"error": err,
-				}).Debug("Failed to augment with CUPS-get-printers request.")
-			}
 		}
 	}
 
+	if resp.StatusCode != 200 {
```

Comment:
Same comment on status code as above, except this is more problematic, since it won't even let you get to the partial responses (will it?)